19:33:57 [ERROR] Error when accessing content:
Traceback (most recent call last):
  File "/mnt/d/Programmation/Rust/projects/webserver-rs/data/pages/post/login.py", line 1, in <module>
    from lib.scripting_utils import *
ModuleNotFoundError: No module named 'lib'

